src/pages/LoginToAdminPage.js

```diff
@@ -56,6 +56,7 @@ function LoginToAdminPage() {
         };
         if (result.status === 403) {
             setShowPasswordErrorMessage(true)
+            sessionStorage.clear(password);
```

Comment:
This is ok because we have only one item in the sessionStorage right now.
But we should delete a specific key. `clear` removes everything in the sessionStorage.
```
sessionStorage.removeItem('key');
```